src/react-input-placeholder.js

```diff
@@ -141,16 +141,25 @@ var createShimmedElement = function(React, elementConstructor, name) {
     });
 };
 
-module.exports = function(React) {
+module.exports = function(React, ReactDom) {
+    var _domNode;
+    if (React.findDOMNode) {
```

Comment:
I'm thinking we should reduce the complexity a bit and just always require ReactDOM... what do you think?